SignalUtilitiesKit/Profile Pictures/Identicon+ObjC.swift

```diff
@@ -8,7 +8,17 @@ public final class Identicon : NSObject {
         if content.count > 2 && content.hasPrefix("05") {
             content.removeFirst(2)
         }
-        let layer = icon.generateLayer(with: size, text: content.substring(to: 1))
+        let initials: String = content
+            .split(separator: " ")
+            .compactMap { word in word.first.map { String($0) } }
+            .joined()
+        let layer = icon.generateLayer(
+            with: size,
+            text: (initials.count >= 2 ?
+                initials.substring(to: 2).uppercased() :
+                content.substring(to: 2).capitalized
```

Comment:
Just thinking if we still want to capitalized if the original text includes 2 capital letters. 🤔